client/components/single-product.js

```diff
@@ -14,9 +24,9 @@ class SingleProduct extends React.Component {
     quantity: 1
   }
 
-  componentDidMount() {
+  async componentDidMount() {
```

Comment:
looks like this isn't necessary; product info updates without async call since the fetchProduct function itself is async await